armi/utils/tests/test_utils.py

```diff
@@ -250,6 +251,20 @@ def test_convertToSlice(self):
         with self.assertRaises(Exception):
             slice1 = utils.convertToSlice("slice")
 
+    def test_plotMatrix(self):
+        fname = "F"
```

Comment:
I recommend using a more unique filename here. We run into trouble in parallel with os.remove locks sometimes in the off chance some other tests makes a file called that. Maybe `test_plotMatrix_testfile.png`?

Reply:
Other option is to do the whole thing in a TemporaryDirectoryChanger, then we dont need to worry about removing anything.